src/core/image.js

```diff
@@ -271,6 +339,19 @@ var PDFImage = (function PDFImageClosure() {
       }
     }
 
+    if ((resizeWidth > 0 && resizeWidth < width) &&
```

Comment:
we will require the resizeWidth and resizeHeight to have a value, see comment above. let's use `resizeWidth < width || resizeHeight < height`to decide if we need to downsize.